templates/sources.list.ubuntu.deb822.tmpl

```diff
@@ -0,0 +1,60 @@
+## template:jinja
```

Comment:
I don't think it's worth defining a separate fully commented entry because multiple `Types:` can be declared on one line in a given entry.

That said, in ubuntu.deb822.tmpl, I had the commented `# Types: deb deb-src` above our active `Types: deb` in the top entry, but is was missing from the security source entry.

```diff
diff --git a/templates/sources.list.ubuntu.deb822.tmpl b/templates/sources.list.ubuntu.deb822.tmpl
index 7b65d53c2..d75dc088f 100644
--- a/templates/sources.list.ubuntu.deb822.tmpl
+++ b/templates/sources.list.ubuntu.deb822.tmpl
@@ -53,6 +53,7 @@ Signed-By: /usr/share/keyrings/ubuntu-archive-keyring.gpg
 
 ## Ubuntu security updates. Aside from URIs and Suites,
 ## this should mirror your choices in the previous section.
+# Types: deb deb-src
 Types: deb
 URIs: {{security}}
 Suites: {{codename}}-security
```

In sources.list.debian.deb822.tmpl, I left both `Types: deb deb-src` active because that's what debian has done in packaging in the original sources.list template. If we think that's worth changing, we can make debian align w/ ubuntu and avoid updating deb-src repos.